packages/react-components/react-link/src/components/Link/useLinkStyles.styles.ts

```diff
@@ -101,6 +111,7 @@ export const useLinkStyles_unstable = (state: LinkState): LinkState => {
     appearance === 'subtle' && styles.subtle,
     inline && styles.inline,
     disabled && styles.disabled,
+    backgroundAppearance === 'inverted' && styles.inverted,
```

Comment:
wont the inverted color override the disabled color style?

Reply:
yes it will, good catch!